server/utils.ts

```diff
@@ -94,7 +94,11 @@ export const getRelativeTime = (oldTimestamp: number) => {
 const MAX_RETRIES = 10
 
 const delay = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms))
-
+export const isAbortError = (error: any) =>
```

Comment:
We won't modify for now, but if the error is specific to a app and entity then we need to ensure that is documented somewhere.
In future we can do this in a mapper way for errors across apps.
No need to change much for now other than document the error app.